deps/softwarecontainer-dependencies.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+# vagrant-cookbook
+# Copyright (C) 2015 Pelagicore AB
+#
+# Permission to use, copy, modify, and/or distribute this software for
+# any purpose with or without fee is hereby granted, provided that the
+# above copyright notice and this permission notice appear in all copies.
+#
+# THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL
+# WARRANTIES WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED
+# WARRANTIES OF MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL THE AUTHOR
+# BE LIABLE FOR ANY SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES
+# OR ANY DAMAGES WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS,
+# WHETHER IN AN ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION,
+# ARISING OUT OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS
+# SOFTWARE.
+#
+# For further information see LICENSE
+#
+#
+# Usage: softwarecontainer-dependencies.sh
+#
+# Installs dependencies for softwarecontainer
+#
+
+echo "Installing softwarecontainer dependencies"
+
+function install {
+    packages="$@"
+
+    retval=100
+    count=0
+
+    if [[ "$retval" -ne "0" && $count -le 5 ]]; then
+        count=$count+1
+        DEBIAN_FRONTEND=noninteractive apt-get -o Dpkg::Options::="--force-confnew" --force-yes -fuy install $packages
+        retval=$?
+    fi
+    if [[ "$retval" -ne "0" ]]; then
+        exit $retval
+    fi
+}
+
+# For softwarecontainer
+install libdbus-c++-dev libdbus-1-dev libglibmm-2.4-dev libglibmm-2.4 \
```

Comment:
I'm wondering about the stable vs testing stuff, should we have some kind of if testing install this, else if stable install this package as a convention? For the package which changes name?